bin/jmeter.bat

```diff
@@ -151,25 +149,21 @@ if not defined HEAP (
     set HEAP=-Xms1g -Xmx1g -XX:MaxMetaspaceSize=256m
 )
 
-rem Uncomment this to generate GC verbose file with Java prior to 9
-rem set VERBOSE_GC=-verbose:gc -Xloggc:gc_jmeter_%%p.log -XX:+PrintGCDetails -XX:+PrintGCCause -XX:+PrintTenuringDistribution -XX:+PrintHeapAtGC -XX:+PrintGCApplicationConcurrentTime -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCDateStamps -XX:+PrintAdaptiveSizePolicy
-
-rem Uncomment this to generate GC verbose file with Java 9 and above
-rem set VERBOSE_GC=-Xlog:gc*,gc+age=trace,gc+heap=debug:file=gc_jmeter_%%p.log
-rem You may want to add those settings
-rem -XX:+ParallelRefProcEnabled -XX:+PerfDisableSharedMem
+rem Legacy GC verbose options removed (Java 8/9 support discontinued)
+rem Enhanced G1GC tuning for Java 17
+rem Improved G1 performance with better defaults and new features
 if not defined GC_ALGO (
-    set GC_ALGO=-XX:+UseG1GC -XX:MaxGCPauseMillis=100 -XX:G1ReservePercent=20
+    set GC_ALGO=-XX:+UseG1GC -XX:MaxGCPauseMillis=100 -XX:G1ReservePercent=20 -XX:+ParallelRefProcEnabled -XX:+UseStringDeduplication -XX:G1HeapRegionSize=16m
```

Comment:
`ParallelRefProcEnabled` seems to be enabled by default as well

Reply:
Thanks for comment, change in last PR version